ckanext/datastore/config_declaration.yaml

```diff
@@ -92,6 +92,16 @@ groups:
       them. It can be overwritten by the user by passing the "lang" parameter to
       "datastore_search" and "datastore_create".
 
+  - default: 60_000
+    key: ckan.datastore.default_query_timeout
```

Comment:
I might even add `_ms` to this name because without reading the docs users won't know the units.

Alternatively make this a float value in seconds because all the other config values are in seconds.